src/link/constants.scss

```diff
@@ -71,7 +71,7 @@ $link-variants: (
     'text-color-default': awsui.$color-text-link-default,
     'text-color-hover': awsui.$color-text-link-hover,
     'text-color-active': awsui.$color-text-link-hover,
-    'font-weight': awsui.$font-button-weight,
+    'font-weight': awsui.$font-weight-button,
```

Comment:
don't think that this should inherit the button weight: info link is visually styled more like a link not a button

Reply:
Resolved!